<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1881</width>
    <height>1122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_ctrl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>321</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>控制单元</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>301</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>模式</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_server">
         <property name="text">
          <string>服务器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_local">
         <property name="text">
          <string>本地</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_test">
         <property name="text">
          <string>测试</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_start">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string> 系统启动</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_end">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string> 系统关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_top">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>拓扑关系配置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_link_end">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>440</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>链路测试结束</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_link_start">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>链路参数测试</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sim_start">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>开始仿真</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sim_end">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>340</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>结束仿真</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_readserver">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>接入服务器</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>751</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>运行日志</string>
    </property>
    <widget class="QTextEdit" name="textEdit_log">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>731</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_display">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>1011</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>显示单元</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>991</width>
       <height>581</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>    场景</string>
      </attribute>
      <widget class="QLabel" name="label_scene">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>5</y>
         <width>918</width>
         <height>540</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>动态显示</string>
      </attribute>
      <widget class="QLabel" name="label_m3">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>270</y>
         <width>30</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_6">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>300</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m7">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>270</y>
         <width>30</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m2">
       <property name="geometry">
        <rect>
         <x>285</x>
         <y>270</y>
         <width>30</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m11">
       <property name="geometry">
        <rect>
         <x>565</x>
         <y>360</y>
         <width>30</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_7">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>300</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m5">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>270</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_3">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>300</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_8">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>300</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m8">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>270</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m9">
       <property name="geometry">
        <rect>
         <x>245</x>
         <y>360</y>
         <width>30</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_12">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>390</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m6">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>270</y>
         <width>30</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_1">
       <property name="geometry">
        <rect>
         <x>175</x>
         <y>310</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_11">
       <property name="geometry">
        <rect>
         <x>525</x>
         <y>390</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_4">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>300</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_9">
       <property name="geometry">
        <rect>
         <x>205</x>
         <y>390</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_ap2">
       <property name="geometry">
        <rect>
         <x>575</x>
         <y>130</y>
         <width>200</width>
         <height>150</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_10">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>390</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_5">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>300</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_ap1">
       <property name="geometry">
        <rect>
         <x>245</x>
         <y>160</y>
         <width>180</width>
         <height>150</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m10">
       <property name="geometry">
        <rect>
         <x>365</x>
         <y>360</y>
         <width>30</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m4">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>270</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m1">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>270</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_m12">
       <property name="geometry">
        <rect>
         <x>675</x>
         <y>360</y>
         <width>30</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_top_2">
       <property name="geometry">
        <rect>
         <x>245</x>
         <y>310</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_broad">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>881</width>
         <height>550</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <zorder>label_broad</zorder>
      <zorder>label_m3</zorder>
      <zorder>label_top_6</zorder>
      <zorder>label_m7</zorder>
      <zorder>label_m2</zorder>
      <zorder>label_m11</zorder>
      <zorder>label_top_7</zorder>
      <zorder>label_m5</zorder>
      <zorder>label_top_3</zorder>
      <zorder>label_top_8</zorder>
      <zorder>label_m8</zorder>
      <zorder>label_m9</zorder>
      <zorder>label_top_12</zorder>
      <zorder>label_m6</zorder>
      <zorder>label_top_1</zorder>
      <zorder>label_top_11</zorder>
      <zorder>label_top_4</zorder>
      <zorder>label_top_9</zorder>
      <zorder>label_ap2</zorder>
      <zorder>label_top_10</zorder>
      <zorder>label_top_5</zorder>
      <zorder>label_ap1</zorder>
      <zorder>label_m10</zorder>
      <zorder>label_m4</zorder>
      <zorder>label_m1</zorder>
      <zorder>label_m12</zorder>
      <zorder>label_top_2</zorder>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_link_test">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>0</y>
      <width>551</width>
      <height>961</height>
     </rect>
    </property>
    <property name="title">
     <string>链路参数测试</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>500</width>
       <height>300</height>
      </rect>
     </property>
     <property name="title">
      <string>时延</string>
     </property>
     <widget class="QCustomPlot" name="widget_delay" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>470</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>500</width>
       <height>300</height>
      </rect>
     </property>
     <property name="title">
      <string> 误码</string>
     </property>
     <widget class="QCustomPlot" name="widget_error" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>470</width>
        <height>260</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>650</y>
       <width>500</width>
       <height>300</height>
      </rect>
     </property>
     <property name="title">
      <string>  带宽</string>
     </property>
     <widget class="QCustomPlot" name="widget_band" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>470</width>
        <height>260</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_segment_conf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>1341</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>   节点信息</string>
    </property>
    <widget class="QComboBox" name="comboBox_sendPing">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>节点一</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>节点二</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>节点三</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>节点四</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>着陆器</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>实验室</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_receivePing">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <item>
      <property name="text">
       <string>节点一</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>节点二</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>节点三</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>节点四</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>着陆器</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>实验室</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;连接到&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_pingInfo">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>411</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pingTest">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
